public/main.css

```
@@ -25,3 +25,101 @@
         margin-block-start: 3rem;
     }
 }
+
+@layer components {
+    .mp-cards-grid {
+        display: grid;
+        list-style: none;
+        padding-inline-start: 0;
+        column-gap: 24px; /* TODO */
+        row-gap: 16px;
+        justify-content: center;
+        grid-template-columns: repeat(auto-fill, minmax(345px, 1fr));
+    }
+
+    .mp-card {
+        grid-template-columns: [portrait-start] max(125px, 35%) [portrait-end content-start] auto [content-end];
+        column-gap: 8px;
+        background-color: #fff;
+        font-family: Courier New, system-ui, sans-serif;
+        position: relative;
+        border-block-start: 10px solid var(--party-color, #c0c0c0);
```

```suggestion
        border-block-end: 10px solid var(--party-color, #c0c0c0);
```

I feel like this looks better overall - I think it was in one of Alina's designs, but I'm open to disagreement. Otherwise LGTM.

I'll reach out and see what she thinks as well.